cmake/schemes/url_sha1_zlog.cmake.in

```diff
@@ -0,0 +1,51 @@
+# This is configuration file, variable @SOME_VARIABLE_NAME@ will be substituted during configure_file command
```

Comment:
Hi @je-vv, can you explain why you needed a new scheme here instead of using one of the existing schemes? Thanks.

Reply:
zlog doesn't use cmake for building, just plain make.  Therefore there was the need to use its own scheme.

Reply:
How about the `autotools` scheme, which is used for `make`-based projects? https://github.com/cpp-pm/hunter/blob/master/cmake/schemes/url_sha1_autotools.cmake.in

Reply:
AFAIK, there's no autotools support either.  A makefile is all there is to play with.    I wouldn't know how to make it work as if using autotools.  I can cancel this PR, and perhaps someone can provide the autotools version of it.

Reply:
Okay, can you rename this scheme to `url_sha1_make_only.cmake.in`?

Reply:
Done !

Reply:
Thanks!